src/main/webapp/styles/main.css

```diff
@@ -1,49 +1,96 @@
+:root {
+    --primaryLight: #e6f3ff;
+    --primaryDark: #42a5f5;
+    --secondaryDark: #181818;
+    --secondaryLight: #ffffff;
+    --alternativeLight: #e31566;
```

Comment:
These are the new colour variables that @Tom-Goring suggested. They are not compiling in Netbum 8.2, but are in IntelliJ. Let me know if it's working for you.

Reply:
Not sure why they would be a problem at all - the CSS files should just be deployed straight into the WAR archive. What error gets spit out when Netbeans fails? A maven build error?

Reply:
It's just displaying syntax errors in the IDE itself, and then fails to provide any colour changes in the software.

Unexpected semi-colon etc etc.

The rest of the CSS is working as expected.

Reply:
Can confirm css variables dont work in the java

Reply:
My login page looks like this:
![image](https://user-images.githubusercontent.com/26003386/104609372-e735e580-567a-11eb-92dd-39a806456d5d.png)
Is this the intended look - to confirm if CSS is working fully or not.